contentcuration/contentcuration/frontend/channelEdit/pages/StagingTreeView/index.vue

```diff
@@ -1,229 +1,248 @@
+                  <Diff :value="resourcesCountDiff" class="font-weight-bold">
+                    <template slot-scope="{ sign, value }">
+                      ({{ sign }}{{ value ? value : '-' }})
+                    </template>
+                  </Diff>
+                </span>
+                <span class="pl-2" data-test="bottom-bar-stats-file-size">
+                  {{ $tr('totalSize') }}:
+                  <span class="font-weight-bold">{{ formatFileSize(fileSizeStaged) }}</span>
+                  <Diff :value="fileSizeDiff" class="font-weight-bold">
+                    <template slot-scope="{ sign, value }">
+                      ({{ sign }}{{ value ? formatFileSize(value) : '-' }})
+                    </template>
+                  </Diff>
+                </span>
+              </VFlex>
+
+              <VFlex class="bottom-bar-btns">
+                <VDialog
+                  v-model="displaySummaryDetailsDialog"
+                  width="500"
                 >
-                  {{ $tr('openSummaryDetailsDialogBtn') }}
-                </VBtn>
-              </template>
-
-              <VCard data-test="summary-details-dialog">
-                <VCardTitle primary-title class="title font-weight-bold">
-                  {{ $tr('summaryDetailsDialogTitle') }}
-                </VCardTitle>
-                <VCardText>
-                  <DiffTable :stagingDiff="stagingDiff" />
-                </VCardText>
-                <VDivider />
-                <VCardActions>
-                  <VSpacer />
-                  <VBtn
-                    color="primary"
-                    @click="displaySummaryDetailsDialog = false"
-                  >
-                    {{ $tr('closeSummaryDetailsDialogBtn') }}
-                  </VBtn>
-                </VCardActions>
-              </VCard>
-            </VDialog>
-
-            <VDialog
-              v-model="displayDeployDialog"
-              width="500"
-            >
-              <template v-slot:activator="{ on }">
-                <VBtn
-                  color="primary"
-                  data-test="display-deploy-dialog-btn"
-                  v-on="on"
+                  <template v-slot:activator="{ on }">
+                    <VBtn
+                      flat
+                      data-test="display-summary-details-dialog-btn"
+                      v-on="on"
+                    >
+                      {{ $tr('openSummaryDetailsDialogBtn') }}
+                    </VBtn>
+                  </template>
+
+                  <VCard data-test="summary-details-dialog">
+                    <VCardTitle primary-title class="title font-weight-bold">
+                      {{ $tr('summaryDetailsDialogTitle') }}
+                    </VCardTitle>
+                    <VCardText>
+                      <DiffTable :stagingDiff="stagingDiff" />
+                    </VCardText>
+                    <VDivider />
```

Comment:
Might need to double check with design on this, but other modals don't have a divider above the actions. Might want to remove for consistency
![image](https://user-images.githubusercontent.com/7447496/88349358-acbf4180-cd04-11ea-9ab4-fb415a51b1a1.png)
![image](https://user-images.githubusercontent.com/7447496/88349391-c6f91f80-cd04-11ea-83b7-2d57d5bfe5ad.png)